common/autoconf/configure

```diff
@@ -123,10 +123,10 @@ fi
 ###
 # Add default OpenJ9 closed extensions
 full_script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-OPENJDK_TAG=`git -C $full_script_dir/../.. describe --abbrev=0 --tags`
+OPENJDK_TAG=`git -C $full_script_dir/../.. describe --abbrev=0 --tags --match "jdk8u*"`
```

Comment:
JDK9 contains `--match "jdk-9*" "${OPENJDK_SHA}"`. Does the SHA also need to be specified here?

Reply:
I don't think it is needed; git assumes 'HEAD', but I'll add it for consistency.